draft-ietf-rtcweb-jsep.xml

```diff
@@ -1379,37 +1360,11 @@ Suggested:
             callee actually accepts the call, the application can plug
             in the real MediaStreamTrack and create a new "sendrecv"
             offer to update the previous offer/answer pair and start
-            bidirectional media flow. While this could also be done
-            with a "sendonly" pranswer, followed by a "sendrecv"
-            answer, the initial pranswer leaves the offer/answer
-            exchange open, which means that the caller cannot send an
-            updated offer during this time.
-
-<!-- [rfced] Section 4.1.8.1:  We had trouble with this sentence.
-If neither suggestion below is correct, please clarify.
-
-Original:
- While
- this could also be done with a "sendonly" pranswer, followed by a
- "sendrecv" answer, the initial pranswer leaves the offer-answer
- exchange open, which means that the caller cannot send an updated
- offer during this time.
-
-Suggestion #1:
- While
- this could also be done with a "sendonly" pranswer, if followed by a
- "sendrecv" answer the initial pranswer leaves the offer/answer
- exchange open, which means that the caller cannot send an updated
- offer during this time.
-
-Suggestion #2:
- While
- this could also be done with a "sendonly" pranswer followed by a
- "sendrecv" answer, the initial pranswer leaves the offer/answer
- exchange open, which means that the caller cannot send an updated
- offer during this time. -->
-
-</t>
+	    bidirectional media flow.  While this could also be done 
```

Comment:
Indentation seems to have changed. Is this intentional?

Reply:
Are indentation changes an issue?  I just cut and paste suggestion 2 from the comments into the text.  Indents didn't quite align.

Reply:
For ease of reading/reviewing, I'd prefer to maintain the original indent. I can handle this with a post-PR commit if needed.

Reply:
Please fix, thanks!